### ISSUE TITLE
dataset problem

### ISSUE THREAD

**Issue by ifei17-hfut** [dangeng/flowmag#1]
Your work is very innovative! But I'm having a little problem, like how should the generated dataset be obtained? I cannot find dataroot and any way to collect 145k unlabeled frame pairs from several public datasets by myself. Can you provide some help? Thanks a lot!

**Comment by zhaoyingpan** [comment]
Thanks for your interest! We will release the code of the dataset collection soon. You may also find details of dataset collection in our supplementary material, and a zip file of our test images as an example: https://drive.google.com/file/d/1e9KljPpIHB5Yq8r2-XcHLEHlym6n1H5C/view?usp=drive_link

**Comment by ifei17-hfut** [comment]
Thank you very much for your reply! Your work is highly anticipated.

**Comment by ifei17-hfut** [comment]
> Thanks for your interest! We will release the code of the dataset collection soon. You may also find details of dataset collection in our supplementary material, and a zip file of our test images as an example: https://drive.google.com/file/d/1e9KljPpIHB5Yq8r2-XcHLEHlym6n1H5C/view?usp=drive_link

I have reviewed the updated dataset section, and the dataset size is quite large! It took me two to three days to execute the subset scripts. I have a few questions to ask you: Regarding the generate_<dataset name>_subset.py files, I executed them one by one. Since the TAO and Vimeo datasets are large, I tried adjusting the batch_size in the get_loader function to speed up the execution (the original setting was 1). However, I noticed that the resulting JSON files contain multiple pairs of image paths under each index. Will this affect the subsequent execution of collect_subsets.py? If it does, I need to immediately stop the process and revert the batch_size to 1. I look forward to your response!
